quetz/cli.py

```diff
@@ -132,13 +132,13 @@ def _make_migrations(
     if plugin_name == "quetz":
         version_path = None  # Path(quetz.__file__).parent / 'migrations' / 'versions'
     else:
-        entry_point = next(
-            pkg_resources.iter_entry_points('quetz.migrations', plugin_name)
-        )
+        entry_point = tuple(
```

Comment:
does the `next` trick not work anymore?

Reply:
It is but I think it would raise a deprecation warning. It's recommended to cast to tuple instead

Reply:
Who recommends that? It could hypothetically use a lot more space etc